src/token.scss

```diff
@@ -109,7 +109,7 @@ $block: #{$fd-namespace}-token;
     line-height: 1.125rem;
 
     .#{$block}__close {
-      padding: $fd-token-compact-padding-vertical $fd-token-compact-padding-horizontal;
+      padding: ($fd-token-compact-padding-vertical * 2) $fd-token-compact-padding-horizontal $fd-token-compact-padding-vertical;
```

Comment:
I'm not sure it's a good idea to change the paddings. You can easily fix the issue by changing the flex `alig-items` rule to normal:
<img width="241" alt="Screen Shot 2021-04-28 at 12 19 22 PM" src="https://user-images.githubusercontent.com/39598672/116438161-2ea44400-a81c-11eb-9e22-e3a261c54767.png">

Reply:
You will get this:
<img width="697" alt="Screen Shot 2021-04-28 at 12 21 14 PM" src="https://user-images.githubusercontent.com/39598672/116438231-42e84100-a81c-11eb-9787-496e9b54cd67.png">

Reply:
@InnaAtanasova done in a bit different way. Please re-check.